evap/staff/forms.py

```diff
@@ -178,7 +179,7 @@ def clean(self):
         vote_start_date = self.cleaned_data.get('vote_start_date')
         vote_end_date = self.cleaned_data.get('vote_end_date')
         if vote_start_date and vote_end_date:
-            if vote_start_date >= vote_end_date:
+            if vote_start_date.date() >= vote_end_date:
```

Comment:
this was the point where we discussed whether to allow evaluations starting and ending on the same day, right? was there any reason to not allow that? if you change this (`>=` to `>`), please do so in a separate commit.